lib/ts/recipe/session/with-jwt/sessionClass.ts

```diff
@@ -56,7 +56,37 @@ export default class SessionClassWithJWT implements SessionContainerInterface {
         return this.originalSessionClass.getExpiry(userContext);
     };
 
-    updateAccessTokenPayload = async (newAccessTokenPayload: any, userContext?: any): Promise<void> => {
+    assertClaims(claimValidators: SessionClaimValidator[], userContext?: any): Promise<void> {
+        return this.originalSessionClass.assertClaims.bind(this)(claimValidators, userContext);
+    }
+
+    applyClaimBuilder = <T>(claimBuilder: SessionClaimBuilder<T>, userContext?: any) => {
+        return this.originalSessionClass.applyClaimBuilder.bind(this)(claimBuilder, userContext);
+    };
+
+    setClaimValue = <T>(claimBuilder: SessionClaimBuilder<T>, value: T, userContext?: any) => {
+        return this.originalSessionClass.setClaimValue.bind(this)(claimBuilder, value, userContext);
+    };
+
+    removeClaim = (claimBuilder: SessionClaimBuilder<any>, userContext?: any) => {
+        return this.originalSessionClass.removeClaim.bind(this)(claimBuilder, userContext);
+    };
+
+    mergeIntoAccessTokenPayload = async (accessTokenPayloadUpdate: any, userContext?: any): Promise<void> => {
+        const updatedPayload = { ...this.getAccessTokenPayload(userContext), ...accessTokenPayloadUpdate };
+        for (const key of Object.keys(accessTokenPayloadUpdate)) {
+            if (accessTokenPayloadUpdate[key] === null) {
+                delete updatedPayload[key];
+            }
+        }
+
+        await this.updateAccessTokenPayload(updatedPayload, userContext);
```

Comment:
If we call the original implementation with bind(this), then that will call the `updateAccessTokenPayload` function from this class which will update the access token payload that is JWT aware?

Reply:
Right, I misread this; looking at it in more context you mean the entire `mergeIntoAccessTokenPayload` function instead of just line 87.
You are correct, it should call through, in an earlier implementation there was some diff but not anymore.